projects/atlant.md

```diff
@@ -0,0 +1,22 @@
+## Overview
+![logo](../projects/logo/atlant.png)  
+Name: ATLANT   
+[Website](https://atlant.io/)  
+Founded: 2016  
+Currency: ATL  
+ICO: Q4 2017
```

Comment:
ICO End date is even more important, because https://keycryptovc.slack.com/archives/C6BP9497A/p1503336474000295
and
https://keycryptovc.slack.com/archives/C6BP9497A/p1503409506000494

ICO End date: 31. Oct 2017
 (https://tokenmarket.net/blockchain/ethereum/assets/atlant/)